app/nhs-health-check-online/2025/11/managing-live-updates/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Designing for change: how we’re managing live updates in our service"
+date: 2025-11-01
```

Comment:
sorry – can you make this today’s date on Monday‘s (whenever you get to publishing it